bin/dockerfilelint

```diff
@@ -4,14 +4,36 @@ var process = require('process');
 var fs = require('fs');
 var os = require("os");
 var path = require('path');
-var argv = require('yargs').argv;
+var usage = 'Usage: dockerfilelint [files | content..] [options]';
+var argv = require('yargs')
+             .usage(usage)
```

Comment:
@alecjacobs5401 Hey, were you planning on changing the indentation? It's really no big deal, just wasn't sure exactly what you meant. Once we settle this, I'll approve the PR.